SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditor.cpp

```diff
@@ -136,13 +253,13 @@ bool FSpatialGDKEditor::LoadPotentialAssets(TArray<TStrongObjectPtr<UObject>>& O
 		{
 			return false;
 		}
-		Progress.EnterProgressFrame(1, FText::FromString(FString::Printf(TEXT("Loading %s"), *Data.AssetName.ToString())));
 		if (auto GeneratedClassPathPtr = Data.TagsAndValues.Find("GeneratedClass"))
 		{
 			const FString ClassObjectPath = FPackageName::ExportTextPathToObjectPath(*GeneratedClassPathPtr);
 			const FString ClassName = FPackageName::ObjectPathToObjectName(ClassObjectPath);
 			FSoftObjectPath SoftPath = FSoftObjectPath(ClassObjectPath);
-			OutAssets.Add(TStrongObjectPtr<UObject>(SoftPath.TryLoad()));
+			Progress.EnterProgressFrame(1.f, FText::FromString(FString::Printf(TEXT("Loading %s"), *Data.AssetName.ToString())));
```

Comment:
This seems like the progress bar will not "finish" if there's any assets that aren't generated classes.